mne/preprocessing/ica.py

```diff
@@ -2038,12 +2055,12 @@ def _detect_artifacts(ica, raw, start_find, stop_find, ecg_ch, ecg_score_func,
 
 @verbose
 def run_ica(raw, n_components, max_pca_components=100,
-            n_pca_components=64, noise_cov=None, random_state=None,
-            picks=None, start=None, stop=None, start_find=None,
-            stop_find=None, ecg_ch=None, ecg_score_func='pearsonr',
-            ecg_criterion=0.1, eog_ch=None, eog_score_func='pearsonr',
-            eog_criterion=0.1, skew_criterion=-1, kurt_criterion=-1,
-            var_criterion=0, add_nodes=None, verbose=None):
+            n_pca_components=64, method='fastica', noise_cov=None,
```

Comment:
this is an API break. You should put the method parameter at the end, just before verbose otherwise it's likely to break code.